app/Bus/Commands/Schedule/CreateScheduleCommand.php

```diff
@@ -70,7 +70,7 @@ final class CreateScheduleCommand
         'message'      => 'nullable|string',
         'status'       => 'required|int|min:0|max:2',
         'scheduled_at' => 'required|string',
-        'completed_at' => 'nullable|string',
+        'completed_at' => 'required|string',
```

Comment:
I don't think this makes sense. Maintenance could (and should) be scheduled ahead of time, so the completed time can't be known.

Reply:
But this PR is also cool and would break if it is `nullable`: https://github.com/CachetHQ/Cachet/pull/3520 What is your opinion on that?

Reply:
I think we should revisit that bit of the system then, because it clearly isn't making sense :)

Reply:
Personally we use `completed_at` as a maximum completion time for the maintenance window.